.github/workflows/check-release.yaml

```diff
@@ -99,120 +24,14 @@ jobs:
       - uses: r-lib/actions/setup-r@v2
         with:
           use-public-rspm: true
-          
-      - uses: r-lib/actions/setup-renv@v2
-        with:
-          profile: '"shiny"'
-          
+
```

Comment:
add this here again:
```bash
      - uses: r-lib/actions/setup-renv@v2
        with:
          profile: '"shiny"'
  
      - name: install texlive
        run : |
            sudo apt-get install texlive
            sudo apt-get install texlive-fonts-extra
      - name: R CMD CHECK inside r
        run : |
            install.packages("devtools")
            renv::restore()
            install.packages("mlr3")
            install.packages("mlr3learners")
            install.packages("mlr3measures")
            install.packages("mlr3tuning")
            install.packages("mlr3misc")
            devtools::install_github("mlr-org/mlr3extralearners")
            devtools::install_github("mlr-org/mlr3proba")
            devtools::install_version("mlr3viz", version = "0.5.10")
            rcmdcheck::rcmdcheck(error_on="warning")
        shell : Rscript {0}

```

Reply:
ok

Reply:
we should add `"."` to `path` argument of `devtools::build(path=".")` in **lastMod.sh**

Reply:
not yet working

Reply:
i mark this as resolved, but continue in other comments section